tests/tests.js

```diff
@@ -81,7 +71,7 @@ describe('Discord Tests', function() {
          * Test that the hook returns a 200 when posted to
          */
         it('Hook returns a 200 OK when posted to', function(done) {
-            sendPost('1', function(error, response, body) {
+            sendPost(testUtils.recordedFixturesDir + '1', function(error, response, body) {
```

Comment:
This smells stinky? What does a test directory have to do with sending a POST request?